github_actions/lib/dependabot/github_actions/file_updater.rb

```diff
@@ -65,17 +65,37 @@ def updated_workflow_file_content(file)
             gsub(/@.*+/, "@#{new_req.fetch(:source).fetch(:ref)}")
 
           # Replace the old declaration that's preceded by a non-word character
-          # and followed by a whitespace character (comments) or EOL
+          # and followed by a whitespace character (comments) or EOL.
+          # If the declaration is followed by a comment, attempt to update
+          # any version comments associated with SHA source refs.
           updated_content =
             updated_content.
             gsub(
```

Comment:
you could use `updated_content.gsub!` instead of `updated_content = updated_content.gsub`. The `!` mutates the original `updated_content`